ushahidi/platform-client-mzima
# USH-1024 Removing pre-selection when adding a new data-source

This PR:
- Removes the preselected datasource in the dropdown when adding a new datasource
- Displays the spinner when waiting for the data

To test:
- Go to settings->data-sources
- Click on "Add source"
- The spinner appears for a while when fetching data (not entirely sure why it takes so long? Or if it just on my side?)
- The dropdown appears, with no preselection

When testing, make sure that:
- The edit-form behaves as expected. It is using the same code as the add-source functionality.

Issue: https://linear.app/ushahidi/issue/USH-1024/on-opening-add-data-source-page-dont-preselect-a-data-source-fe
